Apps/ValidationTests/macOS/ViewController.mm

```diff
@@ -69,10 +70,14 @@ - (void)initialize {
     engineView.autoresizingMask = NSViewWidthSizable | NSViewHeightSizable;
     [[self view] addSubview:engineView];
     engineView.delegate = engineView;
-
+    
     void* windowPtr = (__bridge void*)engineView;
 
-    graphics = Babylon::Graphics::CreateGraphics(windowPtr, static_cast<size_t>(600), static_cast<size_t>(400));
+    GraphicsConfiguration graphicsConfig = GraphicsConfiguration();
+    graphicsConfig.windowPtr = engineView;
```

Comment:
How does this compile? Isn't `graphicsConfig.windowPtr` of type `MTKView*`?

Reply:
we're declaring the interface `EngineView` to be an `MTKView` via L18